torchaudio/models/wav2vec2/utils/import_huggingface.py

```diff
@@ -32,11 +32,17 @@ def _get_config(cfg):
 
 
 def _build(config, original):
+    if original.__class__.__name__ == 'Wav2Vec2ForCTC':
```

Comment:
possible to perform the comparison on classes directly rather than string class names?

Reply:
We could, but the intention here is to avoid `import transformers` inside of `torchaudio` library, so that we do not have `transformers` as an optional dependency, taking advantage of duck-typing.

Of course the expectation is that `transformers` is installed and imported on client code, so it should be practically no problem to do it.

Reply:
got it — sounds reasonable